codebases/index.html

```diff
@@ -48,6 +48,16 @@ <h1 class="title">Stage 2: <strong>Incubation</strong></h1>
 
   <div class="cb-listings">
 
+    <div class="cb-listing">
+      <!-- div class="cb-logo"><img src="/photos/codebases/open-source-project-template/open-source-project-template.png" alt="Open Source Project Template logo"></div -->
+      <div class="cb-info">
+        <div class="cb-name">Open Source Project Template</div>
+        <div class="cb-desc">Open Source project helper and get-going template(s)</div>
```

Comment:
'Getting started' is more conventional than 'get-going'. Was there a reason you chose this?
```suggestion
        <div class="cb-desc">Open Source project helper and getting started template(s)</div>
```

Reply:
This is the short-text description from the github repository.

We can certainly suggest changes to them.